strongbox-security/strongbox-user-management/src/main/resources/etc/conf/strongbox-security-users.yaml

```diff
@@ -28,11 +18,11 @@ users:
                 repositoryPrivileges:
                   - name: ARTIFACTS_DEPLOY
                   - name: ARTIFACTS_RESOLVE
-    - username: developer01
+    - username: test-user
```

Comment:
We shouldn't have this test user in our production file. One option is to have the test create this user and use it as needed.

Reply:
OK I have removed it. How can I create this user on the test env?

Reply:
I believe you should create these users programatically and set them up via the respective set up method.